tools/bin/load_test/load_test_airbyte.sh

```diff
@@ -0,0 +1,252 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+
+<<comment
+This script performs a load test against an existing Airbyte instance by calling the instance's API to create and sync new connections.
+It is intended to work with any Airbyte instance (local or remote, docker or kube, OSS or Cloud). It authenticates using a special auth header
+that requires port-forwarding. It stores connector and connection IDs that it creates in a local file. The script can be run in cleanup mode,
+which means the script will delete every connector and connection ID that it created and stored in that file.
+comment
+
+cd "$(dirname "$0")"
+. load_test_utils.sh
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;34m'
+CLEAR='\033[0m'
```

Comment:
Do we need to redefine this? Does `load_test_utils.sh` already load these in? Might be my lack of bash understanding.